Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -175,6 +175,379 @@ public void TestExecuteAsyncWithMaxRetryReached()
                 Assert.GreaterOrEqual(stopwatch.ElapsedMilliseconds, 30 * 1000);
             }
         }
+
+        [Test]
+        public void TestAsyncExecQueryAsync()
+        {
+            string queryId;
+            var expectedWaitTime = 5;
+            Task task;
+
+            using (SnowflakeDbConnection conn = new SnowflakeDbConnection())
+            {
+                conn.ConnectionString = ConnectionString;
+                task = conn.OpenAsync(CancellationToken.None);
+                task.Wait();
```

Comment:
if you would change the signature of the test method to `public async Task TestAsyncExecQueryAsync()` then you could write just await conn.OpenAsync(CancellationToken.None).ConfigureAwait(false);
Maybe it would be a little bit shorter.

Reply:
Done. I'll do the same for other async tests

I was following how other async tests did it so I wasn't sure which one was preferred. I'll do it this way from now on